test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -109,49 +110,103 @@ baz
         $actual.Length | Should Be $tailCount
         $actual[0] | Should Be $expected
     }
-    It "should Get-Content with a variety of -Tail and -ReadCount values" {#[DRT]
-        Set-Content -Path $testPath "Hello,World","Hello2,World2","Hello3,World3","Hello4,World4"
-        $result=Get-Content -Path $testPath -Readcount:-1 -Tail 5
-        $result.Length | Should Be 4
-        $expected = "Hello,World","Hello2,World2","Hello3,World3","Hello4,World4"
-        for ($i = 0; $i -lt $result.Length ; $i++) { $result[$i]  | Should BeExactly $expected[$i]}
-        $result=Get-Content -Path $testPath -Readcount 0 -Tail 3
-        $result.Length    | Should Be 3
-        $expected = "Hello2,World2","Hello3,World3","Hello4,World4"
-        for ($i = 0; $i -lt $result.Length ; $i++) { $result[$i]  | Should BeExactly $expected[$i]}
-        $result=Get-Content -Path $testPath -Readcount 1 -Tail 3
-        $result.Length    | Should Be 3
-        $expected = "Hello2,World2","Hello3,World3","Hello4,World4"
-        for ($i = 0; $i -lt $result.Length ; $i++) { $result[$i]  | Should BeExactly $expected[$i]}
-        $result=Get-Content -Path $testPath -Readcount 99999 -Tail 3
-        $result.Length    | Should Be 3
-        $expected = "Hello2,World2","Hello3,World3","Hello4,World4"
-        for ($i = 0; $i -lt $result.Length ; $i++) { $result[$i]  | Should BeExactly $expected[$i]}
-        $result=Get-Content -Path $testPath -Readcount 2 -Tail 3
-        $result.Length    | Should Be 2
-        $expected = "Hello2,World2","Hello3,World3"
-        $expected = $expected,"Hello4,World4"
-        for ($i = 0; $i -lt $result.Length ; $i++) { $result[$i]  | Should BeExactly $expected[$i]}
-        $result=Get-Content -Path $testPath -Readcount 2 -Tail 2
-        $result.Length    | Should Be 2
-        $expected = "Hello3,World3","Hello4,World4"
-        for ($i = 0; $i -lt $result.Length ; $i++) { $result[$i]  | Should BeExactly $expected[$i]}
-        $result=Get-Content -Path $testPath -Delimiter "," -Tail 2
-        $result.Length    | Should Be 2
-        $expected = "World3${nl}Hello4", "World4${nl}"
-        for ($i = 0; $i -lt $result.Length ; $i++) { $result[$i]  | Should BeExactly $expected[$i]}
-        $result=Get-Content -Path $testPath -Delimiter "o" -Tail 3
-        $result.Length    | Should Be 3
-        $expected = "rld3${nl}Hell", '4,W', "rld4${nl}"
-        for ($i = 0; $i -lt $result.Length ; $i++) { $result[$i]  | Should BeExactly $expected[$i]}
-        $result=Get-Content -Path $testPath -AsByteStream -Tail 10
-        $result.Length    | Should Be 10
-        if ($IsWindows) {
-            $expected =      52, 44, 87, 111, 114, 108, 100, 52, 13, 10
+    It "should Get-Content with a variety of -Tail and -ReadCount values" -TestCases @(
```

Comment:
When using `-TestCases` you should update the name of the test case (`It` statement) to include a parameter so when run, the output is unique for each test case variation.  In this particular case you probably want to add a parameter just for the test case title (as well as give the test name something better):

```
It "Should work with -Tail and -ReadCount: <test>" -TestCases @(
  @{ test = "negative readcount"
        GetContentParams = @(...)
        ...
  }
...
```